pollux/core/src/main/scala/org/hyperledger/identus/pollux/core/service/CredentialServiceImpl.scala

```diff
@@ -1196,31 +1200,36 @@ class CredentialServiceImpl(
 
   private def allocateNewCredentialInStatusListForWallet(
       record: IssueCredentialRecord,
-      statusListRegistryUrl: String,
+      statusListRegistryServiceName: String,
       jwtIssuer: JwtIssuer
   ): URIO[WalletAccessContext, CredentialStatus] = {
     val effect = for {
       lastStatusList <- credentialStatusListRepository.getLatestOfTheWallet
       currentStatusList <- lastStatusList
-        .fold(credentialStatusListRepository.createNewForTheWallet(jwtIssuer, statusListRegistryUrl))(
+        .fold(credentialStatusListRepository.createNewForTheWallet(jwtIssuer, statusListRegistryServiceName))(
           ZIO.succeed(_)
         )
       size = currentStatusList.size
       lastUsedIndex = currentStatusList.lastUsedIndex
       statusListToBeUsed <-
         if lastUsedIndex < size then ZIO.succeed(currentStatusList)
-        else credentialStatusListRepository.createNewForTheWallet(jwtIssuer, statusListRegistryUrl)
+        else credentialStatusListRepository.createNewForTheWallet(jwtIssuer, statusListRegistryServiceName)
       _ <- credentialStatusListRepository.allocateSpaceForCredential(
         issueCredentialRecordId = record.id,
         credentialStatusListId = statusListToBeUsed.id,
         statusListIndex = statusListToBeUsed.lastUsedIndex + 1
       )
+      resourcePath =
+        s"credential-status/${statusListToBeUsed.id}"
+      segment = statusListToBeUsed.lastUsedIndex + 1
     } yield CredentialStatus(
-      id = s"$statusListRegistryUrl/credential-status/${statusListToBeUsed.id}#${statusListToBeUsed.lastUsedIndex + 1}",
+      id =
+        s"""${jwtIssuer.did}?resourceService="$statusListRegistryServiceName"&resourcePath="$resourcePath#$segment"""",
```

Comment:
I think that it should be removed as well.
The [example](https://www.w3.org/TR/did-spec-registries/#service-param) doesn't have "around the service"

Reply:
hmm, I agree on removing. I was thinking if it would be better to use base64URL encoding instead of escaping characters
But I am mostly raising the option, I don't have a strong opinion. I should update the spec though in any case